src/main/java/uk/gov/companieshouse/exemptions/controller/ExemptionsController.java

```diff
@@ -52,7 +52,7 @@ public ResponseEntity<Void> companyExemptionsUpsert(
     public ResponseEntity<CompanyExemptions> companyExemptionsGet(
             @RequestHeader("x-request-id") String contextId,
             @PathVariable("company_number") String companyNumber) {
-        logger.info(String.format("Getting company exemptions for company number %s", companyNumber));
+        logger.info(String.format("KCP - Getting company exemptions for company number %s", companyNumber));
```

Comment:
Leftover from local testing here looks like

Reply:
Nice spot, added this in to help with debugging
Removed in the latest commit.